packages/fetchai/connections/p2p_noise/publickey.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
There are a few things missing:
- encoding, shebang, copyright notice (copy & paste from here: https://github.com/fetchai/agents-aea/blob/master/setup.py#L1-L19)
- docstring that states the purpose of this module

Reply:
file deleted